SoftLayer/managers/event_log.py

```diff
@@ -0,0 +1,38 @@
+"""
+    SoftLayer.event_log
+    ~~~~~~~~~~~~~~~~~
+    Network Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+
+class EventLogManager(object):
+    """Provides an interface for the SoftLayer Event Log Service.
+
+    See product information here:
+    http://sldn.softlayer.com/reference/services/SoftLayer_Event_Log
+    """
+    def __init__(self, client):
+        self.client = client
+
+    def get_event_logs(self, request_filter):
+        """Returns a list of event logs
+
+        :param dict request_filter: filter dict
+        :returns: List of event logs
+        """
+        results = self.client.call("Event_Log",
```

Comment:
From a convention standpoint, usually you can just store the event log service on the manager itself. So you'd add `self.event_log = client['Event_Log']` to `__init__()`, and then in these functions you can just do `self.event_log.getAllObjects(filter=request_filter)`